
--- elisa/core/plugin_registry.py.orig	2009-02-15 13:52:58.000000000 +0100
+++ elisa/core/plugin_registry.py	2009-02-15 13:53:39.000000000 +0100
@@ -156,8 +156,8 @@
     default_config = \
         {'repository': 'http://elisa-plugins.fluendo.com/plugin_list',
          'update_plugin_cache': True,
-         'auto_update_plugins': True,
-         'auto_install_new_recommended_plugins': True}
+         'auto_update_plugins': False,
+         'auto_install_new_recommended_plugins': False}
     config_doc = \
         {'repository': 'The plugin repository to query for new plugins and ' \
             'plugin updates.',

