diff -urN source/org.kde.elisa.desktop source-patched/org.kde.elisa.desktop
--- source/org.kde.elisa.desktop	2017-08-11 05:46:51.000000000 +0300
+++ source-patched/org.kde.elisa.desktop	2017-08-16 10:45:54.358819705 +0300
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Categories=Qt;KDE;Audio;AudioVideo
 Comment=
+Comment[ru]=Музыкальный проигрыватель
 Exec=elisa
 GenericName=Elisa Music Player
 GenericName[ast]=Reproductor de música Elisa
@@ -17,6 +18,7 @@
 GenericName[pl]=Odtwarzacz muzyki Elisa
 GenericName[pt]=Leitor de Música Elisa
 GenericName[pt_BR]=Reprodutor de músicas Elisa
+GenericName[ru]=Музыкальный проигрыватель Elisa
 GenericName[sk]=Hudobný prehrávač Elisa
 GenericName[sv]=Elisa musikspelare
 GenericName[uk]=Музичний програвач Elisa
@@ -38,6 +40,7 @@
 Name[pl]=Elisa
 Name[pt]=Elisa
 Name[pt_BR]=Elisa
+Name[ru]=Elisa
 Name[sk]=Elisa
 Name[sv]=Elisa
 Name[tr]=Elisa
diff -urN source/src/localFileConfiguration/kcm_elisa_local_file.desktop source-patched/src/localFileConfiguration/kcm_elisa_local_file.desktop
--- source/src/localFileConfiguration/kcm_elisa_local_file.desktop	2017-08-11 05:46:51.000000000 +0300
+++ source-patched/src/localFileConfiguration/kcm_elisa_local_file.desktop	2017-08-16 10:43:27.348810148 +0300
@@ -17,6 +17,7 @@
 Name[nl]=Lokale bestanden van Elisa indexeerder
 Name[nn]=Indeksering av lokale filer for Elisa
 Name[pt]=Indexação de Ficheiros Locais do Elisa
+Name[ru]=Индексный файл локальных файлов
 Name[sk]=Indexer miestnych súborov Elisa
 Name[sv]=Elisa lokal filindexering
 Name[uk]=Засіб індексування локальних файлів Elisa
@@ -30,6 +31,7 @@
 Comment[nl]=Een configuratiehulpmiddel voor de indexeerder van lokale bestanden van de muziekspeler Elisa
 Comment[nn]=Oppsettverktøy for indeksering av lokale filer for musikkspelaren Elisa
 Comment[pt]=Uma ferramenta de configuração para a indexação de ficheiros locais no leitor de música Elisa
+Comment[ru]=Настройки для индекса локальных файлов музыкального проигрывателя Elisa
 Comment[sk]=Konfiguračný nástroj pre indexer miestnych súborov hudobného prehrávača Elisa
 Comment[sv]=Ett inställningsverktyg för lokal filindexering i Elisa musikspelare
 Comment[uk]=Інструмент налаштовування засобу індексування локальних файлів музичного програвача Elisa
@@ -43,6 +45,7 @@
 X-KDE-Keywords[nl]=Elisa
 X-KDE-Keywords[nn]=Elisa
 X-KDE-Keywords[pt]=Elisa
+X-KDE-Keywords[ru]=Elisa
 X-KDE-Keywords[sk]=Elisa
 X-KDE-Keywords[sv]=Elisa
 X-KDE-Keywords[uk]=Elisa
diff -urN source/src/localFileConfiguration/package/metadata.desktop source-patched/src/localFileConfiguration/package/metadata.desktop
--- source/src/localFileConfiguration/package/metadata.desktop	2017-08-11 05:46:51.000000000 +0300
+++ source-patched/src/localFileConfiguration/package/metadata.desktop	2017-08-16 10:44:21.313813657 +0300
@@ -14,6 +14,7 @@
 Name[pl]=Elisa
 Name[pt]=Elisa
 Name[pt_BR]=Elisa
+Name[ru]=Elisa
 Name[sk]=Elisa
 Name[sv]=Elisa
 Name[tr]=Elisa
@@ -28,6 +29,7 @@
 Comment[nl]=Indexeerder van lokale bestanden van de muziekspeler Elisa configureren
 Comment[nn]=Set opp indeksering av lokale filer for musikkspelaren Elisa
 Comment[pt]=Configurar a Indexação de Ficheiros Locais no Leitor de Música Elisa
+Comment[ru]=Настройка индекса локальных файлов музыкального проигрывателя Elisa
 Comment[sk]=Nastaviť indexer miestnych súborov hudobného prehrávača Elisa
 Comment[sv]=Anpassa lokal filindexering för musikspelaren Elisa
 Comment[uk]=Налаштувати індексування локальних файлів для музичного програвача Elisa
