diff -Nru a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2018-09-29 21:27:08.000000000 +0200
+++ b/src/CMakeLists.txt	2018-09-30 11:41:46.669072453 +0200
@@ -197,9 +197,9 @@
 
 if (NOT APPLE AND NOT WIN32)
     install(TARGETS elisaLib
-        LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/elisa
-        RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/elisa
-        BUNDLE DESTINATION ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/elisa
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}/elisa
+        RUNTIME DESTINATION ${LIB_INSTALL_DIR}/elisa
+        BUNDLE DESTINATION ${LIB_INSTALL_DIR}/elisa
     )
 else()
     install(TARGETS elisaLib ${INSTALL_TARGETS_DEFAULT_ARGS})
@@ -232,7 +232,7 @@
 
 if (NOT APPLE AND NOT WIN32)
     set_target_properties(elisaqmlplugin PROPERTIES
-        INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/elisa;${CMAKE_INSTALL_RPATH}"
+        INSTALL_RPATH "${LIB_INSTALL_DIR}/elisa;${CMAKE_INSTALL_RPATH}"
     )
 endif()
 
@@ -351,7 +351,7 @@
 
     if (NOT APPLE AND NOT WIN32)
         set_target_properties(elisa PROPERTIES
-            INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/elisa;${CMAKE_INSTALL_RPATH}"
+            INSTALL_RPATH "${LIB_INSTALL_DIR}/elisa;${CMAKE_INSTALL_RPATH}"
         )
     endif()
 
